apps/eew/sceewlog/eews2fcm.py

```diff
@@ -36,31 +35,71 @@ def __init__(self, fcmDataFile, worldCitiesFile, language = "es-US"):
         self.language = language
         self.agency = ''
         self.distance = 0
-    
+        self.projectId = "" 
+        self.serviceAccountFile = ""
+
     def readFcmDataFile(self):
         if os.path.exists( self.fcmDataFile ):
             config = configparser.RawConfigParser()
-            
+
         else:
             seiscomp.logging.error("the file %s does not exists" % self.fcmDataFile )
             return -1
-        
+
         try:
             config.read(self.fcmDataFile)
         except Exception as e:
             seiscomp.logging.error("Not possible to open %s " % self.fcmDataFile)
             return -1
-            
+
         try:
             self.authKey = config.get('AUTHKEY', 'key')
             self.topic = config.get('TOPICS', 'topic')
+            self.serviceAccountFile = config.get('SERVICEFILE', 'servicefile')
+            self.projectId = config.get('PROJECTID', 'projectid')
```

Comment:
please also add these to sceewlog.xml

Reply:
Possible but it makes more sense to have a separate file for only FCM configuration options and calling this from the main configuration file (?). 
Please, keep in mind tokens and service JSON file are different. Using the service JSON file we obtain a short-lived access token to send the notifications (that lasts one hour).

Reply:
Ok I did not get that fcm can provide this entire file. In that case it is totally acceptable. But does that mean the JSON config file will always need to be updated with the long-life token file?